src/main/cpp/blaze_util_bsd.cc

```diff
@@ -184,9 +184,9 @@ blaze_util::Path GetProcessCWD(int pid) {
     procstat_freeprocs(procstat, p);
   }
   procstat_close(procstat);
-  return blaze_util::Path(cwd);
+  return std::unique_ptr<blaze_util::Path>(new blaze_util::Path(cwd));
```

Comment:
I think the blessed idiom these days is to use `std::make_unique`. Not sure that'll pass CI though.

Reply:
I tried `std::make_unique`. The build failed.